Makefile

```diff
@@ -1,40 +1,72 @@
+python-dir=./python/src
+node-dir=./node/src
+
 # Anything that needs to be done before the other rules run
 setup:
 	mkdir -p build/data
 
 # Runs the entire pipeline using real data sources
 release: setup
 	curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$$limit=50000' \
-	  | node parse-trees.js \
-	  | python pruning_planting.py \
-	  | node download-images.js \
-	  | node split-trees.js build/data
+	  | node $(node-dir)/parse-trees.js \
+	  | python $(python-dir)/pruning_planting.py \
+	  | node $(node-dir)/download-images.js \
+	  | node $(node-dir)/split-trees.js build/data
+
+release-docker: setup
+	curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$$limit=50000' \
```

Comment:
This won't work, the proper query only has 1 $

Reply:
![It Runs](https://user-images.githubusercontent.com/13500774/69597260-34a84c80-0fba-11ea-893a-939bd9fff1f7.png)

Please see the command I just ran with the double "$$" in the make file, but the results in the output are as expected.

Why is this the case:

This stack overflow reference should explain why this happens:
https://stackoverflow.com/questions/19589957/makefile-and-use-of#comment29075486_19589957